config/locales/js-en.yml

```diff
@@ -315,24 +315,28 @@ en:
-                <li>The user administration is distributed to non-admins by a <b>new global role</b>, allowing to manage and invite users without having full admin privileges.</li>
-                <li>Further UI and UX improvements, i.e. harmonizing cost reporting navigation, adapted background colors of sum and group rows in the work package list, adding a group show page.</li>
+                <li>New button in header navigation to create projects, users and work packages.</li>
+                <li>A new invite modal for users, groups and placeholder users allows to easily add or invite new users and assign them to work packages.</li>
+                <li>GitHub integration into OpenProject.</li>
+                <li>API v3 extensions for groups and more, i.e. create, read, update and delete groups and group members through the API.</li>
+                <li>Multi-selection for project custom fields of type list.</li>
+                <li>Creation of backup from the web interface.</li>
               </ul>
           bim:
-            learn_about_link:  https://www.openproject.org/openproject-11-2-release
+            learn_about_link:  https://www.openproject.org/openproject-11-3-release
             current_new_feature_html: >
               The release contains various new features and improvements: <br>
               <ul class="%{list_styling_class}">
-                <li>A new concept of <b>placeholder users</b> (premium feature) allows to add preliminary users without email address to OpenProject and to assign tasks to these resources.</li>
-                <li>The work package <b>filters are extended by a parent option </b>to easily see all direct children as well as <b>by ID</b> to include or exclude only these exact work packages in the list.</li>
-                <li>The user administration is distributed to non-admins by a <b>new global role</b>, allowing to manage and invite users without having full admin privileges.</li>
-                <li>Further UI and UX improvements, i.e. harmonizing cost reporting navigation, adapted background colors of sum and group rows in the work package list, adding a group show page.</li>
+                <li>New button in header navigation to create projects, users and work packages.</li>
+                <li>A new invite modal for users, groups and placeholder users allows to easily add or invite new users and assign them to work packages.</li>
+                <li>GitHub integration into OpenProject.</li>
+                <li>API v3 extensions for groups and more, i.e. create, read, update and delete groups and group members through the API.</li>
+                <li>Multi-selection for project custom fields of type list.</li>
+                <li>Creation of backup from the web interface.</li>
```

Comment:
@wielinde, currently BIM has the same text as the standard edition. Do you want to highlight changes in BIM? If so, go ahead and add them right here.

Reply:
@ulferts done.